content/docs/manage/facilities.md

```diff
@@ -0,0 +1,101 @@
++++
+title = "Facilities"
+description = "Manage all stores"
+date = 2022-05-17T16:20:00+00:00
+updated = 2022-05-17T16:20:00+00:00
+draft = false
+weight = 2
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+toc = true
+top = false
++++
+
+The Facilities list is available only on the [Open mSupply Central Server](/docs/getting_started/central). This is where you can view all stores, and manage their properties.
+
+## Viewing Facilities
+
+Choose `Manage` > `Facilities` in the navigation panel.
+
+![goto: facilities](/docs/manage/images/goto_facilities.png)
+
+You will be presented with a list of stores in your system.
+
+![facility list](/docs/manage/images/facilities.png)
+
+The list of Stores is divided into 5 columns:
+
+| Column       | Description                      |
+| :----------- | :------------------------------- |
+| **Code**     | The code for the store           |
+| **Name**     | Store name                       |
+| **Supplier** | Whether this store is a supplier |
+| **Customer** | Whether this store is a customer |
+| **Donor**    | Whether this store is a donor    |
+
+<div>
+You will also see an icon like this (<img src="/docs/replenishment/images/is_msupplystoreicon.png" alt="Store" style="width:auto">) next to the store code - meaning this is an mSupply store.
```

Comment:
```suggestion
You will also see an icon like this (<img src="/docs/replenishment/images/is_msupplystoreicon.png" alt="Store" style="width:auto">) next to the store code - meaning this is an Open mSupply store.
```

though it could be an mSupply store. I think more confusing to have 'mSupply' though, no?

Reply:
I might just take it out - they're all stores, so its there on every row, so it feels less meaningful (compared to supplier list where its there for some and not others?)